AI_Graphics/AI_Graphics/Inspector/Inspector.cs

```diff
@@ -1,8 +1,7 @@
 ﻿using UnityEngine;
 using static AIGraphics.Inspector.Util;
 
-namespace AIGraphics.Inspector
-{
+namespace AIGraphics.Inspector {
```

Comment:
I guess this is unintentional? could we revert?

Reply:
Well my Visual Studio Sometimes does that idk why